components/team_general_tab/team_general_tab.tsx

```diff
@@ -178,17 +184,15 @@ export default class GeneralTab extends React.PureComponent {
         }
     }
 
-    handleClose = () => {
-        this.updateSection('');
-    }
+    handleClose = () => this.updateSection('');
```

Comment:
before the change, the return type of the function was void, then after the change the return type is that of this.updateSection(''). which is not too much of a change, but i would be concerned of these changes in scope of this pr